rules/linux/macos_binary_padding.yml

```diff
@@ -0,0 +1,31 @@
+title: 'Binary Padding'
+id: 95361ce5-c891-4b0a-87ca-e24607884a96
+status: experimental
+description: 'Adversaries may use binary padding to add junk data and change the on-disk representation of malware. This rule detect using dd and truncate to add a junk data to file.'
+    # For this rule to work you must enable audit of process execution in OpenBSM, see
+    # https://osquery.readthedocs.io/en/stable/deployment/process-auditing/#macos-process-socket-auditing
+author: 'Igor Fits, Mikhail Larin, oscd.community'
+date: 2020/10/19
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1027.001/T1027.001.md
+logsource:
+    product: macos
+    category: process_creation
+detection:
+    selection1:
+        CommandLine|contains|all:
+             - 'truncate'
+             - '-s'
+    selection2:
+        CommandLine|contains|all:
+             - 'dd'
+             - 'if='
```

Comment:
Hello @alx1m1k! Could you please processname from the commandline here? It will dramatically improve the performance of the query. Please apply the same logic to the rest of your macOS rules. Thank you!